pygmt/src/meca.py

```diff
@@ -283,6 +283,10 @@ def meca(
 
         # convert dict to pd.DataFrame so columns can be reordered
         if isinstance(spec, dict):
+            # convert values to ndarray so pandas doesn't complain about "all
+            # scalar values". See issue #2174 for details.
```

Comment:
Prefer to put the link to the pull request
```suggestion
            # scalar values". See
            # https://github.com/GenericMappingTools/pygmt/pull/2174
```